core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -2021,6 +2023,137 @@ public static Long toTimestampWithLocalTimeZone(String v, TimeZone timeZone) {
         .getMillisSinceEpoch();
   }
 
+  public static int timeWithTimeZoneToTime(TimeWithTimeZone ttz) {
```

Comment:
Do we need javadoc for these `public static` methods?

Reply:
I think the method name has documented it?